# Define the request
float64[] initial_state
hexapod_control/Pose goal
---
# Define the result
float64[] solution
int32 result
string error_message